functions/zoom-uploader.py

```diff
@@ -108,8 +111,11 @@ def title(self):
 
     @property
     def description(self):
+        formatted_time = datetime.strftime(
```

Comment:
previous suggestion about datetime objs would spare this back and forth parse/unparse.